schemas/reports/definitions.yaml

```diff
@@ -14,29 +14,11 @@ report_shared:
       type: string
       description: The API endpoint to retrieve the report. Read-only.
     status:
-      type: string
-      enum:
-        - awaiting_data
-        - awaiting_approval
-        - cancelled
-        - complete
-        - withdrawn
-      description: The current state of the report in the checking process. Read-only.
+      $ref: "#/report_status"
```

Comment:
Referencing objects this way allows to get a single reusable object for all the reports.

Reply:
Makes sense, but how do we know such changes don't impact the structure of onfido-node?

Reply:
I've tested locally and it seems to work well, without major differences.
Anyway as onfido-node hasn't been delivered yet, we can still update it if needed.